ci-operator/step-registry/openstack/provision/etcd-disk-speed/openstack-provision-etcd-disk-speed-commands.sh

```diff
@@ -9,20 +9,16 @@ function info() {
 }
 
 function wait_for_etcd_patch_done() {
-  info 'Waiting for patch etcd cluster...'
-  for _ in {1..30}; do
-    ETCD_CO_AVAILABLE=$(oc get co etcd | grep etcd | awk '{print $3}')
-    if [[ "${ETCD_CO_AVAILABLE}" == "True" ]]; then
-      echo "Patched successfully!"
-      break
-    fi
-    sleep 15
-  done
-  if [[ "${ETCD_CO_AVAILABLE}" != "True" ]]; then
-          echo "Etcd patch failed..."
-          exit 1
+  info 'INFO: Waiting for patch etcd cluster...'
```

Comment:
Nit: Please remove the INFO prefix, it's already in the output of the info() function (on [line 8](https://github.com/openshift/release/pull/59057/files#diff-61a78a69fc41150f08b8b255492a18d34f09d942945bd0bc0f94ae9656d3bec3R8)).